debugger/src/debug_adapter/dap/adapter.rs

```diff
@@ -792,7 +792,7 @@ impl<P: ProtocolAdapter> DebugAdapter<P> {
                             source: None,
                             instruction_reference: None,
                             offset: None,
-                            verified: true,
+                            verified: false,
```

Comment:
Why is this changed?

Reply:
It was a bug I discovered while testing the breakpoints in C. It is in the Err() and is the opposite of verified :)